src/solana/SolanaEngine.ts

```diff
@@ -765,15 +768,19 @@ export class SolanaEngine extends CurrencyEngine<
     const versionedMessage = MessageV0.compile({
       instructions,
       payerKey: payer,
-      recentBlockhash
+      recentBlockhash: recentBlockhash.blockhash
     })
     const versionedTx = new VersionedTransaction(versionedMessage)
 
     const unsignedTx =
       asSolanaSpendInfoOtherParams(spendInfoOtherParams)?.unsignedTx ??
       versionedTx.serialize()
 
-    const otherParams = wasSolanaTxOtherParams({ unsignedTx })
+    const otherParams = wasSolanaTxOtherParams({
+      unsignedTx,
+      blockhash: '',
+      lastValidBlockHeight: 0
```

Comment:
Since we have it already, we could return `blockhash: recentBlockhash.blockhash,
lastValidBlockHeight: recentBlockhash.lastValidBlockHeight` instead of dummy values. They aren't used, true. Still, might be safer?

Reply:
sure thing, done